public/locales/en-US/dashboard.json

```diff
@@ -193,8 +193,8 @@
       "table_head": {
         "name": "Name",
         "reported_hashrate": "Reported Hashrate",
-        "current_e_hashrate": "Current E. Hashrate",
-        "average_e_hashrate": "Average Effective Hashrate",
+        "current_e_hashrate": "Current Hashrate",
+        "average_e_hashrate": "Average Hashrate",
         "valid": "Valid Shares",
```

Comment:
Better to revert this because it will be unclear if something is about effective or reported hashrate.

Reply:
I can only fit the E. in due to space constraints but they have been added back in.